notificationworkerlambda/src/main/scala/com/gu/notifications/worker/Harvester.scala

```diff
@@ -40,12 +42,12 @@ trait HarvesterRequestHandler extends Logging {
     throwables.broadcastTo(logErrors, cloudwatch.sendFailures(env.stage, platform))
   }
 
-  def platformSink(shardedNotification: ShardedNotification, platform: Platform, deliveryService: SqsDeliveryService[IO]): Sink[IO, (String, Platform)] = {
-    val iosSinkErrors = sinkErrors(platform)
+  def platformSink(shardedNotification: ShardedNotification, platform: Platform, deliveryService: SqsDeliveryService[IO], buildTier: Option[BuildTier] = None): Sink[IO, HarvestedToken] = {
```

Comment:
It seems you aren't using `buildTier` here, which I believe is a good thing! You probably want to drop the parameter 👍